### DISCUSSION TITLE
Saleor 3.0 Webhooks

### DISCUSSION THREAD

**Discussion by Juleszz** [saleor/saleor#8248] [General]
Hey there, 

I was just wondering if webhooks for Saleor 3.0 are currently under development? I could find documentation about version 2.9 but once I open the webhooks section in my plugins there is just nothing listed.
<img width="1209" alt="Bildschirmfoto 2021-09-30 um 15 21 57" src="https://user-images.githubusercontent.com/86893534/135463126-6b7a1834-34ff-4532-8017-67fb43292888.png">

**Comment by karolkielecki** [comment]
@Juleszz Hi! You can find documentation about webhooks here: https://docs.saleor.io/docs/3.0/dashboard/apps#webhooks

To learn more about the webhook protocols, see the Developer's Guide: https://docs.saleor.io/docs/3.0/developer/extending/apps/key-concepts

**Comment by nnthuan** [comment]
@karolkielecki Hi! I have a problem below:

Saleor 3.5.0-a (current main)

I created a local app and created two webhooks for this app (Graphql)
  1 - To receive an ORDER_CREATED event.
  2 - To receive an ORDER_FULLY_PAID event.
Two webhooks have the same target URL.

I create a checkout session and create an order from this checkout (Graphql)
In the dashboard, the order is created, but I can't receive any POST webhooks.

**Reply by nnthuan** [reply]
I found the cause of not receiving the webhook. It's that the local app must have order management permission.
